deploy.sh

```diff
@@ -126,9 +126,19 @@ setup_kessel() {
   echo "Kessel inventory is setting up.."
   bonfire deploy kessel -C kessel-inventory --set-image-tag quay.io/redhat-services-prod/project-kessel-tenant/kessel-inventory/inventory-api=latest
 
+  apply_latest_schema
   setup_sink_connector
 }
 
+apply_latest_schema() {
+  echo "Applying latest SpiceDB schema from rbac-config"
+  curl -o schema.zed https://raw.githubusercontent.com/RedHatInsights/rbac-config/refs/heads/master/configs/stage/schemas/schema.zed
```

Comment:
can we pull the schema inside "deploy" folder? i think that is gitignored just to be safe? WDYT? And remove the same later on

Reply:
Yeah that's a great point, i'll change that up quickly